blackbox/docs/editions/enterprise.rst

```diff
@@ -33,6 +32,12 @@ enterprise features are enabled:
 - `The CrateDB admin UI`_: `shards browser`_, `monitoring overview`_,
   `privileges browser`_
 
+
+.. note::
+
+   It is also possible to build a :ref:`community-edition` which won't contain
+   these features but which can be used without licensing restrictions.
+
 .. _enterprise_trial:
 
 Trial
```

Comment:
Shouldn't we depreacate the Trial as well as it will be removed with 4.0?

Reply:
See my comment here https://github.com/crate/crate/pull/8345#discussion_r263794126  - we'll update this once we've the logic change in place.